var/spack/repos/builtin/packages/py-pynn/package.py

```diff
@@ -30,3 +31,6 @@ class PyPynn(PythonPackage):
 
     depends_on('py-neo@0.3:0.4.1',      type=('build', 'run'), when="@:0.8.3")
     depends_on('py-neo@0.5.0:',         type=('build', 'run'), when="@0.9.0:")
+    depends_on('py-lazyarray',          type=('build', 'run'), when="@0.9.0:")
+
+    depends_on('py-mock@1.0:', type='test')
```

Comment:
There's actually a lot of other packages needed for testing:
```python
        tests_req = ['mpi4py', 'scipy;python_version>="3.4"',                            
                     'matplotlib;python_version>="3.4"', 'Cheetah3',                     
                     'h5py']
```
But Spack doesn't actually run any of these tests right? I think you can remove the test deps.

Reply:
Yes, you are right, I removed the test dependency